### ISSUE TITLE
Any plan for 1.20.1 and updating the docs to the latest?

### ISSUE THREAD

**Issue by PeterMa9621** [mc-tau-ui/Tau#16]
**Describe the feature you'd like to see**
Any plan for 1.20.1 and updating the docs to the latest?

**Describe feature use cases**

**Additional context**

**Comment by winterSteve25** [comment]
Hi, thank you for your interest. There's already a port for 1.20.4 if you check the branches. I'm not entirely sure if it works with 1.20.1 but I think there shouldn't be a big problem. I am however currently not planning a port for 1.20.1 specifically if the 1.20.4 port is not compatible with 1.20.1, if there's interest I can port it but I may or may not be able to maintain feature parody with the upcoming versions, I hope you understand.

The 1.20.4 port is not feature complete yet but all the features that were there in the previous versions should all work fine.

I haven't had time to update documentation yet, but I will hopefully get to it soon. In the meantime, there's not much changed since the previous versions so you can probably refer to the doc and change log to get started for now :)

Also as a last note there are currently plans to update the 1.20 port to 1.20.6 soon.

**Comment by PeterMa9621** [comment]
Thank you for your reply! I tried 1.20.4 but I don't think it woks with 1.20.1 as I think NeoForge is not stable for 1.20.1 so people will mainly develop mobs based on Forge. I checked the source code of 1.20.4, I believe you ported it to NeoForge.

It looks like you implemented UI for inventory (container), is it possible to get an exmaple for it :)?

**Comment by winterSteve25** [comment]
That's correct it is running neoforge for 1.20.4. I can see if I can port it to forge 1.20.1, but like I stated earlier, I wont be able to maintain feature parody with future versions.

**Comment by winterSteve25** [comment]
Hey, I realized that I have already updated neoforge to 1.20.6 for the latest version, so its not on 1.20.4, my mistake. As of the UI Inventory example, I am currently using this library for another project so there will be an example coming up soon, and I will be adding documentation to the website in due time.

**Comment by PeterMa9621** [comment]
Thank you so much!

**Comment by winterSteve25** [comment]
The forge 1.20.1 port can now be found on the 1.20.1forge-dev branch, I haven't really tested it, so there may be some issues. I did not upload it onto curseforge or the maven though so you may have to build it yourself
